extensions/wikia/MercuryApi/MercuryApiHelper.class.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+class MercuryApiHelper {
+
+	/**
+	 * Check if a string is a valid article title
+	 *
+	 * @param string $title
+	 * @return bool
+	 */
+	public static function isTitleStringValid( $title ) {
+		return !is_null( $title ) && trim( $title ) !== '';
```

Comment:
If you're wandering how I got this doubt, it came to me while I was creating this test case: https://github.com/Wikia/app/blob/CONCF-432/extensions/wikia/MercuryApi/tests/MercuryApiHelperTest.php#L50